www/src/components/LayoutFooter/ExternalFooterLink.js

```diff
@@ -22,7 +22,8 @@ const ExternalFooterLink = ({children, href, target}) => (
       },
     }}
     href={href}
-    target={target}>
+    target={target}
+    rel={rel}>
```

Comment:
Some `ExternalFooterLink`s in the `Footer` were passing in a `rel` prop but this component was not making use of them.

Reply:
Nice find.